src/bin/wd_copier.rs

```diff
@@ -44,8 +51,30 @@ fn main() {
         bytes_written += read_size;
     }
 
-    println!("syncing...");
+    progress_bar.set_job_title("syncing... (2/2)");
+
+    let (tx, rx) = mpsc::channel();
+
+    let meminfo = procfs::Meminfo::new().unwrap();
+    let dirty_after_copy = meminfo.dirty - starting_dirty;
+
+    thread::spawn(move || loop {
```

Comment:
If it's not too much of a pain, can the contents of this closure be moved to a separate function?

Reply:
Sure, give me a heads up when you've looked at everything else and I can handle that last. I wanna make sure the base concept here is acceptable

Reply:
I make no promise that I won't request other changes :) The concept seems acceptable though.

Reply:
a562f67